queries.sql

```diff
@@ -8,4 +8,70 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name <> 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN TRANSACTION;
```

Comment:
- [ ] Great work so far in crafting your database queries to align with the project requirements! 👏 However, I'd like to discuss how you initiate your transactions. I noticed that you begin a transaction using both the `BEGIN` and `TRANSACTION` keywords, but it's worth noting that using the `BEGIN` keyword alone is sufficient. When you start a query with `BEGIN` in PostgreSQL, the database automatically recognizes that you're initiating a transaction. Therefore, including the `TRANSACTION` keyword after `BEGIN` is unnecessary and not considered best practice. 😊

Please note that this change should be made across all your queries where you're doing something similar as I have described. 

Happy coding! 😊🚀🍀